manager/src/main/java/io/gingersnapproject/database/DatabaseHandler.java

```diff
@@ -48,20 +48,15 @@ void start(@Observes StartupEvent ignore, Config config) {
    }
 
    public void refreshSchema() {
-      tables = configuration.rules().values().stream()
+      tables = configuration.eagerRules().values().stream()
             .map(rule -> vendor.describeTable(pool, rule.connector().table()).await().indefinitely())
```

Comment:
I think the `DatabaseHandler` also needs to handle Eager rules being added/removed.

On `@Observes  RuleEvents.EagerRuleAdded` we need to add an entry to `tables` and `table2rule`. Similarly, on `@Observes RuleEvents.EagerRuleRemoved` we need to remove the corresponding entries from `tables` and `table2rule`.